dev/EnvironmentManager/ChangeTracker/EnvironmentVariableChangeTracker.cpp

```diff
@@ -42,7 +19,7 @@ namespace winrt::Microsoft::Windows::System::implementation
 
     HRESULT EnvironmentVariableChangeTracker::TrackChange(std::function<HRESULT(void)> callback)
     {
-        if (m_ShouldTrackChange)
+        if (ShouldChangesBeTracked(m_Scope))
```

Comment:
NIT: `m_Scope` isn't camelCase.

`m_scope` is expected, per Coding Guidelines. This is a common issue, and particularly catches the eye after seeing `m_Scope`, `m_Value` etc and then youy eye wanders down to `m_eVValue`...

Maybe not worth doing in this PR but I wouldn't object to seeing a PR that make a bunch of global search/replace...

Reply:
Casing will be in a different PR.